package.json

```diff
@@ -138,14 +138,14 @@
     "babel-plugin-module-resolver": "^5.0.0",
     "clang-format": "^1.6.0",
     "code-tag": "^1.1.0",
-    "eslint": "^8.44.0",
+    "eslint": "^8.55.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-config-standard": "^16.0.3",
+    "eslint-config-standard": "^17.1.0",
     "eslint-import-resolver-babel-module": "^5.3.1",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-jest": "^27.2.1",
+    "eslint-plugin-n": "^16.4.0",
```

Comment:
What's that?

Reply:
`eslint-plugin-node` seems to be no longer maintained and `eslint-plugin-n` is its fork.